data/logbook_data.py

```diff
@@ -22,8 +22,8 @@
         "default": {"dest_file": "Speedvac"},
     },
     "Amplification and Purification": {
-        "lims_instrument": {"dest_file": "PCR"},
-        "udf_Bravo": {"dest_file": "Bravo", "details": ["Processname"]},
+        "lims_instrument": {"dest_file": "Bravo", "details": ["Processname"]},
+        "udf_PCR Machine": {"dest_file": "PCR"},
```

Comment:
The space between PCR and Machine bugs me a bit. But  I guess it's fine

Reply:
Hi, `PCR Machine` is the UDF name in LIMS. We don't have a naming convention to avoid space there unfortunately...